participant/src/lib.rs

```diff
@@ -0,0 +1,79 @@
+use frost::{Error, Identifier};
+use frost_ed25519 as frost;
+use std::io::BufRead;
+
+#[derive(Debug, PartialEq)]
+pub struct Config {
+    identifier: Identifier,
+}
+
+pub trait Logger {
+    fn log(&mut self, value: String);
+}
+
+pub fn request_inputs(input: &mut impl BufRead, logger: &mut dyn Logger) -> Result<Config, Error> {
+    logger.log("Your identifier:".to_string());
```

Comment:
```suggestion
    logger.log("Your identifier (an integer between 1 and 65535, inclusive):".to_string());
```

Reply:
The identifier isn't something the participant chooses here, it's sent from the trusted dealer so I don't know if we need to instruct them on what to enter. Maybe it could be more like `this should be an integer between 1 and 65535`.

Reply:
Added in [b82f938](https://github.com/ZcashFoundation/frost-zcash-demo/pull/22/commits/b82f93861fc903f6c47ed6c1168334a10bb92e97)

Reply:
Ah of course :man_facepalming: Yep that looks good, thanks!